manual/config/enterprise/roles_permissions.md

```diff
@@ -10,7 +10,7 @@ NOTE: Admin privileges are not part of the user role.
 
 ## Available Permissions
 
-SeaTable version 2.3 supports 17 permissions:
+SeaTable supports the following permissions since Version 2.6:
```

Comment:
If a Role Permission is removed in a specific version, the table should reflect that. 

Please add a "Removed in version" column to the table and fill in the version when it got removed.

And if the number of supported permissions is to be documented and there are changes since version 2.3, IMHO the sentence should be kept original and extended for the more specific number since the version 2.6 to not loose the old information.